containers/images/core/container-assets/database.yml

```diff
@@ -0,0 +1,12 @@
+production: &default
+  adapter: postgresql
+  encoding: unicode
+  database: postgres
+  pool: 5
+  username: postgres
```

Comment:
Wouldn't this need to be overridden at runtime?

Not sure how it works now, but relying on `DATABASE_URL` has worked well for ManageIQ.